docs/scenarios_authoring/Spel.md

```diff
@@ -264,18 +264,96 @@ can be accessed in e.g. in following ways:
 * `#exampleObjects['someNestedObject']['someFieldInNestedObject']`
 * `#exampleObjects['someArrayWithObjects'][0]['someFieldInObjectInArray']`
 
-Every unknown accessed field/element will produce `Unknown` data type which can be further navigated or [cast](#Casting) to a desired type.
+Every unknown accessed field/element will produce `Unknown` data type, which can be further navigated or [converted](#type-conversions) to a desired type.
 
 ### Type conversions
 
-It is possible to convert from a type to another type and this can be done by implicit and explicit conversion.
+It is possible to convert or cast from a type to another type and this can be done by implicit and explicit conversion.
+
+#### Explicit conversions
+
+Explicit conversions/casts are available as built-in functions.
+List of built-in functions:
+- `canBe(className)`/`to(className)`/`toOrNull(className)`
+- `canBeBoolean`/`toBoolean`/`toBooleanOrNull`
+- `canBeLong`/`toLong`/`toLongOrNull`
+- `canBeDouble`/`toDouble`/`toDoubleOrNull`
+- `canBeBigDecimal`/`toBigDecimal`/`toBigDecimalOrNull`
+- `canBeList`/`toList`/`toListOrNull`
+- `canBeMap`/`toMap`/`toMapOrNull`
+
+The aforementioned functions first attempt to cast a value to the specified class. If the cast fails and there is a 
+defined conversion to that class, the conversion is applied.
+The `canBe`, `to` and `toOrNull` functions take the name of target class as a parameter, in contrast to, for
+example, `canBeLong` which has the name of target class in the function name. 
```

Comment:
Let's explain why we have both `canBe` and these variants with target class in the function name by adding some addition to this sentence. For example ... and is the shortcut fro `canBe('Long')`